index.js

```diff
@@ -448,12 +448,13 @@ const plugin = fp(async function (app, opts) {
       operationName
     )
 
-    return maybeFormatErrors(execution, reply)
+    return maybeFormatErrors(execution, context)
   }
 
-  function maybeFormatErrors (execution, reply) {
+  function maybeFormatErrors (execution, context) {
```

Comment:
@mcollina could you update the index.d.ts definition of `errorFormatter`? Very coincidental timing on your PR but I'm _just_ now trying out/setting up fastify-gql and wanted to access the context from my error logging. :-) Thanks!

Reply:
If you have time to send a PR, it'd be quicker.